### ISSUE TITLE
[ Accessibility] Color Contrast in "Settings > Users" section

### ISSUE THREAD

**Issue by kivuvarosekivuvan** [ushahidi/platform#4971]
# Engineers Checklist
### Describe the bug
The table heading ( "Name") in the "Settings > Users" section does not have enough contrast between the text and the background color, making it difficult to read.

### To Reproduce
1. Launch the local host http://localhost:4200/  and log in to the platform to have "Settings" feature
2. Navigate to the setting section http://localhost:4200/settings/
3. Scroll down to ''Users" section
4. Test the contrast using any accessibility testing tool. For our case we used WAVE

### Expected behavior
The table heading should have enough contrast with the background. 

### How to solve
Increase the contrast between the foreground (text) color and the background color of the table heading. The Web Content Accessibility Guidelines (WCAG) recommend a minimum contrast ratio of 4.5:1 for normal text and 3:1 for large text (larger than 18 points or 14 points bold).

### Screenshots
![Screenshot from 2024-06-12 08-18-03](https://github.com/ushahidi/platform-client-mzima/assets/124133577/e0d73fdb-c675-4c34-86f1-a0efa2a7458c)

[Screencast from 12-06-2024 08:15:00 ASUBUHI.webm](https://github.com/ushahidi/platform-client-mzima/assets/124133577/ee016584-579f-4bb8-8b7c-038aee114818)

**Comment by Angamanga** [comment]
@reuben Could you take a look at this issue and suggest another colour for the heading?

**Comment by reuben** [comment]
@Angamanga this is not the Reuben you are looking for 😄

**Comment by Angamanga** [comment]
@reuben Oh sorry about that! My comment synced from our internal tool and tagged you automatically 😅
